src/scandeval/benchmark_modules/hf.py

```diff
@@ -709,8 +714,20 @@ def get_model_repo_info(
     hf_api = HfApi(token=token)
     model_id, revision = model_id.split("@") if "@" in model_id else (model_id, "main")
 
+    # Check for local model directory
+    model_info = None
+    if os.path.isdir(model_id):
+        logger.debug(f"Checking for local model in {model_id}.")
+        if all(
+            os.path.exists(os.path.join(model_id, f))
+            for f in LOCAL_MODELS_REQUIRED_FILES
+        ):
+            model_info = HfApiModelInfo(id=model_id, tags=None, pipeline_tag=None)
```

Comment:
Change to `pathlib` to maintain consistency with the rest of the codebase:

```suggestion
    model_info = None
    if Path(model_id).is_dir():
        logger.debug(f"Checking for local model in {model_id}.")
        if all(
            (Path(model_id) / f).exists()
            for f in LOCAL_MODELS_REQUIRED_FILES
        ):
            model_info = HfApiModelInfo(id=model_id, tags=None, pipeline_tag=None)
```